c++/src/capnp/rpc.c++

```diff
@@ -3052,6 +3099,44 @@ private:
       return kj::mv(flowPromise);
     }
 
+    kj::Promise<void> sendRealtimeInternal() {
+      // We don't use setupSend() here because we don't actually allocate a question table entry
+      // for realtime messages, because we don't expect a response.
+
+      // Realtime streams do not allow capabilities. We check that there are none and throw
+      // an exception otherwise.
+      if (capTable.getTable().size() != 0) {
+        kj::Exception e = KJ_EXCEPTION(FAILED, "Realtime streams do not allow capabilities!");
+        kj::throwRecoverableException(kj::mv(e));
+      }
+
+      // Finish and send.
+      QuestionId questionId;
+      auto& question = connectionState->questions.nextOneWay(questionId, true);
+      // We don't keep the question for realtime messages. Let's erase it right
```

Comment:
Wouldn't it be better not to have added the question in the first place? There could be a `nextRealtime()` method that returns an incrementing ID without actually inserting anything into the map.